packages/chain-helpers/src/blockchain/Blockchain.utils.spec.ts

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright (c) 2018-2022, BOTLabs GmbH.
+ *
+ * This source code is licensed under the BSD 4-Clause "Original" license
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * @group unit/blockchain
+ */
+
+/* eslint-disable dot-notation */
+import { Keyring } from '@kiltprotocol/utils'
+import type { IIdentity, SubscriptionPromise } from '@kiltprotocol/types'
+import { ApiMocks } from '@kiltprotocol/testing'
+import { setConnection } from '../blockchainApiConnection/BlockchainApiConnection'
+import {
+  IS_FINALIZED,
+  isRecoverableTxError,
+  parseSubscriptionOptions,
+  submitSignedTx,
+} from './Blockchain.utils'
+
+let api: any
+
+beforeAll(() => {
+  api = ApiMocks.getMockedApi()
+  setConnection(Promise.resolve(api))
+})
+
+describe('BlockchainUtils', () => {
```

Comment:
I am uncertain about the file name for the test. Actually even the module BlockchainUtils could be renamed to Blockchain now.